src/applications/static-pages/sass/modules/_m-facilities.scss

```diff
@@ -364,4 +364,28 @@ div.field-vet-center-feature-content  {
   li {
     margin-bottom: 0px
   }
+}
+
+div.field-vet-center-feature-content  {
```

Comment:
We may want to try removing the `div.` in case you want to use a different container, but this works for now.

In the future, if we find ourselves needing to do this elsewhere, we should pick a more generic name for this style.

Reply:
alright